modules/programs/neomutt-accounts.nix

```diff
@@ -0,0 +1,49 @@
+{ config, lib, ... }:
+
+with lib;
+
+{
+  options.neomutt = {
+
+    externalMta = mkOption {
+      type = types.bool;
+      default = true;
+      description = "Use external smtp.";
+    };
+
+    queryCommand = mkOption {
+      type = types.nullOr types.str;
+      default = "notmuch-addrlookup --mutt '%s'";
```

Comment:
The default seems to assume that notmuch is used. Perhaps best to leave as `null`?

Reply:
I removed the setting since I removed its use as well already. I will readd it in a future PR probably.